app/assets/javascripts/modernizr.js

```diff
@@ -1,3 +1,19 @@
+/* 
```

Comment:
This one should probably be removed as well.  It already specifies it's header as MIT & BSD.